### PR TITLE
Better UCR documentation

### PR DESCRIPTION
**Pull request by sascala**
https://jira.mesosphere.com/browse/MARATHON_EE-1161

### CODE REVIEW

File: docs/docs/native-docker.md [hunk 1]
```diff
@@ -1,23 +1,70 @@
...
-# Running Docker Containers on Marathon
+A containerizer is a Mesos agent component responsible for launching containers, within which you can run a Marathon app. Running apps in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to run container images with two different runtimes:
 
-  1. [Docker containerizer](# Docker Containerizer) using the native Docker Engine as runtime.
-  2. Mesos containerizer using the Universal Container Runtime.  
+1. Mesos containerizer using the [Universal Container Runtime](#ucr).
+1. [Docker containerizer](#docker-containerizer) using the native Docker Engine as runtime.
```
Comment (janisz) [inline docs/docs/native-docker.md]: 1 → 2

Comment (sascala) [inline docs/docs/native-docker.md]: Thanks for looking this over, @janisz -- markdown will render lists in numerical order if you start each item with 1. This way, if the list is modified, you don't have to renumber.

File: docs/docs/native-docker.md [hunk 2]
```diff
@@ -1,17 +1,64 @@
...
-# Running Docker Containers on Marathon
+A containerizer is a Mesos agent component responsible for launching containers, within which you can run a Marathon app. Running apps in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to run container images with two different runtimes:
 
-  1. [Docker containerizer](# Docker Containerizer) using the native Docker Engine as runtime.
-  2. Mesos containerizer using the Universal Container Runtime.
+1. Mesos containerizer using the [Universal Container Runtime](#ucr).
+1. [Docker containerizer](#docker-containerizer) using the native Docker Engine as runtime.
```
Comment (Gilbert88) [inline docs/docs/native-docker.md]: s/1/2/g ?

File: docs/docs/native-docker.md [hunk 3]
```diff
@@ -1,17 +1,64 @@
...
 ---
 
-Marathon supports Docker and Appc container images by either one of two runtimes; Docker Engine and Universal Container Runtime.
+# Provisioning Containers
 
-# Running Docker Containers on Marathon
+A containerizer is a Mesos agent component responsible for launching containers, within which you can run a Marathon app. Running apps in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to run container images with two different runtimes:
```
Comment (Gilbert88) [inline docs/docs/native-docker.md]: how about s/run container images/launch containers using container images/g ?

File: docs/docs/native-docker.md [hunk 4]
```diff
@@ -1,17 +1,64 @@
...
+
+The Mesos containerizer provides a `credential`, with a `principal` and an optional `secret` field to authenticate when downloading the Docker image.
+
+```json
+{  
+   "id":"mesos-docker",
+   "container":{  
+      "docker":{  
+         "image":"mesosphere/inky",
+         "credential":{
```
Comment (Gilbert88) [inline docs/docs/native-docker.md]: This does not seem correct. The credential field is deprecated. We use a new secret field called `config` to provide image pulling credentials. Please check with @ichernetsky or @jdef on the Marathon API.

File: docs/docs/native-docker.md (hunk 2 above)
Comment (sascala) [inline docs/docs/native-docker.md]: markdown will render this list correctly -- if you start lists all with 1, then if you have to add something, you don't have to redo all the numbers.

File: docs/docs/native-docker.md [hunk 5]
```diff
@@ -1,17 +1,69 @@
...
+   "cpus":0.2,
+   "mem":16.0,
+   "instances":1
+}
+```
+
+**Important:** If you leave the `args` field empty, the default entry point will be the launch command for the container. If your container does not have a default entry point, you must specify a command in the `args` field. If you do not, your app will fail to deploy.
+
+## UCR Limitations
+- The UCR does not support the following: runtime privileges, Docker options, force pull, named ports, numbered ports, bridge networking, port mapping.
```
Comment (jdef) [inline docs/docs/native-docker.md]: AFAIK we're shipping support for bridge networking and port mappings in Marathon 1.5.

I'm also pretty sure that Marathon supports forced image pull for MESOS containers using a docker image. Is the limitation mentioned here due to a mesos implementation bug? @Gilbert88 

Not sure how "numbered ports" or "named ports" made this list either... should probably be removed. Those are Marathon'ish things, not Mesos things.

**Pull request merged**